lakefs_provider/links/lakefs_link.py

```diff
@@ -0,0 +1,41 @@
+from airflow.models import BaseOperatorLink, XCom
+from airflow.models.baseoperator import BaseOperator
+from airflow.models.taskinstance import TaskInstance, TaskInstanceKey
+from airflow.utils.context import Context
+
+import logging
+
+
+log = logging.getLogger(__name__)
+
+
+LAKEFS_COMMIT_LINK = "{base_url}/repositories/{repo}/commits/{commit_digest}"
```

Comment:
Not done: it looks more like the Google automl links and other extra links in the core Airflow distribution.  I prefer to be like Airflow code, it might be easier to read for someone coming from Airflow.